.circleci/config.yml

```diff
@@ -121,10 +115,7 @@ jobs:
       - checkout
       - *restore_yarn_cache
       - *run_yarn
-      - run:
-          environment:
-            RELEASE_CHANNEL: stable
-          command: yarn test-www --maxWorkers=2
+      - run: yarn test --release-channel=www-classic --ci
 
   RELEASE_CHANNEL_stable_yarn_test_www_variant:
```

Comment:
@acdlite should we rename this tests to `yarn_test_www_classic_variant` and similar for others?

Reply:
Yeah I was also hoping we could express the in terms of the new matrix jobs feature: https://circleci.com/blog/circleci-matrix-jobs/

Reply:
That can be in a follow-up though. Doesn’t need to block.